deploy/tsconfig.json

```diff
@@ -1,10 +1,8 @@
 {
   "compilerOptions": {
-    "target": "ES2018",
+    "target": "ESNext",
```

Comment:
Cutting edge :) I hope it doesn't break on the next "next", instead of a specific pin :)

Reply:
Fair enough. I was using something that required newer typescript features, but ES2021 should be good enough for that.